hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +22,15 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_login_redirects_if_already_logged_in(self):
+        self.client.post("/accounts/check_token/alice/secret-token/")
+
+        r = self.client.post("/accounts/check_token/alice/secret-token/")
```

Comment:
It is login in alice since on the setup we created a token for her which we will be using to log her in so thats why the client  post method is called twice.      
the first we login alice without checking for response and then we login alice again checking for reponse to assert she has been logged in and it redirects to `/checks/`.